pkg/scanner/local/scan.go

```diff
@@ -158,6 +158,9 @@ func (s Scanner) ScanTarget(ctx context.Context, target types.ScanTarget, option
 		s.vulnClient.FillInfo(results[i].Vulnerabilities)
 	}
 
+	// Fill package location information
+	fillPackageLocation(results)
```

Comment:
`Package` already contains location - https://github.com/aquasecurity/trivy/blob/317792433e4e961441f772c6bd22d63873a8c986/pkg/fanal/types/artifact.go#L101

what if we just add location when creating a new vulnerability?
https://github.com/aquasecurity/trivy/blob/317792433e4e961441f772c6bd22d63873a8c986/pkg/detector/library/detect.go#L33-L37

Reply:
> Package already contains location -
> 
> [trivy/pkg/fanal/types/artifact.go](https://github.com/aquasecurity/trivy/blob/317792433e4e961441f772c6bd22d63873a8c986/pkg/fanal/types/artifact.go#L101)
> 
> Line 101 in [3177924](https://github.com/aquasecurity/trivy/commit/317792433e4e961441f772c6bd22d63873a8c986)
> 
>  Locations []Location `json:",omitempty"`
Not clear on this should I create a new type for Locations?

Reply:
No. You can use `ftypes.Location` in `VulnerabilityDetected`.

You have already chosen right solution:
https://github.com/aquasecurity/trivy/blob/d1945dd4e3fe8355621ed2aea65d411efd7fc71e/pkg/types/vulnerability.go#L22